code/modules/jobs/job_types/head_of_security.dm

```diff
@@ -49,7 +49,7 @@
 	suit_store = /obj/item/gun/energy/e_gun
 	r_pocket = /obj/item/assembly/flash/handheld
 	l_pocket = /obj/item/restraints/handcuffs
-	backpack_contents = list(/obj/item/melee/baton/loaded=1, /obj/item/club=1)
+	backpack_contents = list(/obj/item/melee/baton/loaded=1)
```

Comment:
If you had any consistency, you'd lower backpack size as well.

Reply:
Why?